tests/Browser/UserSearchTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Tests\Browser;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Tests\Browser\Pages\UserSearchPage;
+use Tests\Browser\Pages\HomePage;
+use Tests\Browser\Pages\UserPage;
+use Rogue\Models\Signup;
+
+class UserSearchTest extends DuskTestCase
+{
+    /**
+     * Test visiting the User Search Page.
+     * Searching for a user by name (instead of
+     * valid email address) returns "No user found!" message.
+     */
+    public function testIncorrectUserSearchPage()
+    {
+        // Create a signup so that the campaign over page will load.
+        $signup = factory(Signup::class)->create();
+
+        $this->browse(function (Browser $browser) {
+            $browser->visit(new HomePage)
+                    ->click('@login-button')
+                    ->assertPathIs('/register')
+                    ->clickLink('Log In')
+                    ->type('username', 'clee@dosomething.org')
```

Comment:
@DFurnes right now, I'm testing with my own account and have my password in the `NORTHSTAR_PASSWORD=` env variable. I'm not sure how to do this since the tests won't pass without a valid user in Northstar QA right now so I don't think a factory will work. 

I'm thinking for being, I can also create a `NORTHSTAR_EMAIL` env variable in `.env.dusk.local` that devs can input their own credentials to get tests to pass? And for the wercker tests, we can also add a test one? 

OR (this might be better) to just create a test account that is an admin that we can all use across the board? 

Would love your thoughts on this!

Reply:
I think having people enter their own account is a good idea! 👍